apps/backend/src/transaction/payment.service.ts

```diff
@@ -78,12 +78,12 @@ export class PaymentService {
         status,
         transaction: {
           location_id,
-          fiscal_close_date: LessThanOrEqual(date)
+          fiscal_close_date: LessThanOrEqual(date!)
         }
       },
```

Comment:
TO DO: fiscal_close_date less than or equal to date (current deposit date) and greater than or equal to (previous deposit date? or, fiscal_start_date in the reconciliation event?)